shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -873,9 +873,12 @@ private void sendLocalesToFlutter(@NonNull Configuration config) {
       // TODO(garyq) implement a real locale resolution.
       platformResolvedLocale =
           Locale.lookup(languageRanges, Arrays.asList(Locale.getAvailableLocales()));
+
+      flutterEngine.getLocalizationChannel().sendLocales(locales, platformResolvedLocale);
+      return;
```

Comment:
Hm, I dislike these return branches since they tend to introduce bugs when people refactor (like moving this if block above the previous one). Can we simply move the languageRanges definition inside and leave platformResolvedLocale null if we cannot come up with one? That seems easier to grok.

Reply:
Yeah, was trying to fully encapsulate/blocking this logic as I was planning on rewriting it at a future point. But we can just do the simple move for now.